www/src/scss/_theme-common.scss

```diff
@@ -0,0 +1,11 @@
+$highlightStregnth: 0.8;
```

Comment:
this is for highlighting single lines rather than adding an overlay on the whole background. Something that I am trying in sync with aman veerji. To have a general purpose custom background theme option that works with all the themes.

Reply:
typo: Stregnth

Reply:
And it doesn't seem to be used anywhere, or am I missing it? :)

Reply:
It was used in supporting all the themes (when we were trying to use a different way to highlight the verses). I will remove it from this PR and keep it separate for the support of more themes.